services/movie.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
```

Comment:
https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint
Please get familiar with Optional and fix all over the project

Reply:
thank you very much)